ci/test_integration_espnetez.sh

```diff
@@ -17,6 +17,14 @@ gen_dummy_coverage(){
 #### Make sure chainer-independent ####
 python3 -m pip uninstall -y chainer
 
+# First uninstall all espnet-related dependencies including all extras.
+# I use toml and load the pyproject.toml
+python3 -m pip install toml
+python3 test_utils/uninstall_extra.py
```

Comment:
this line is uninstalling nltk, because it is an optional package. Check if the package should become a permanent package or remove the test.

Reply:
Or maybe disable the test related to the package.

Reply:
nvm, I just found the solution

Reply:
@Fhrozen Sonce NOTK is now used in many tasks, I think we can simply move it to the main dependency?

Reply:
Also it will fix the CI issue?